README.md

```diff
@@ -42,6 +42,18 @@ return array(
 );
 ```
 
+The '::class' constant is not available before PHP 5.5. Under a PHP before 5.5 the mapping file can look like this:
+
+```
+<?php
+return array(
+    'Tx_About_Controller_AboutController' => '\TYPO3\CMS\About\Controller\AboutController',
+    'Tx_About_Domain_Model_Extension' => '\TYPO3\CMS\About\Domain\Model\Extension',
```

Comment:
please use quoted backslashes and remove the leading slash:

`TYPO3\\CMS\\About\\Domain\\Model\\Extension`